src/ap-jira-integration/config.json

```diff
@@ -1,23 +1,24 @@
 {
```

Comment:
Please remove these Jira config changes also; shouldn't be part of the PR.

Reply:
Also, found that this PR deletes an image from the GitHub integration. Please resolve this also. Thanks!